### ISSUE TITLE
Source-Stripe: migrate to new integration tests account

### ISSUE THREAD

**Issue by roman-yermilov-gl** [airbytehq/airbyte#21486]
Need to populate data for our integration testing account.

Streams with some records:
```
balance_transactions
customers
payouts
```

Empty streams:
```
bank_accounts
external_account_bank_accounts
charges
checkout_sessions
checkout_sessions_line_items
coupons
customer_balance_transactions
disputes
events
invoice_items
invoice_line_items
invoices
payment_intents
plans
products
promotion_codes
refunds
subscription_items
subscriptions
transfers
```

**Comment by roman-yermilov-gl** [comment]
Switched to test account. Looks like we still don't have enough data.
Streams with some records:
```
balance_transactions
charges
customers
invoice_items
invoice_line_items
invoices
payment_intents
payouts
products
record
```

Empty streams:
```
bank_accounts
external_account_bank_accounts
checkout_sessions
checkout_sessions_line_items
coupons
customer_balance_transactions
disputes
events
plans
promotion_codes
refunds
subscription_items
subscriptions
transfers
```

To compare with current account:
| Streams                       | Records |
| :---------------------------- |:------------ |
| balance_transactions          | 1717 records |
| bank_accounts                 | 2 records    |
| charges                       | 1418 records |
| checkout_sessions             | 2 records    |
| checkout_sessions_line_items  | 2 records    |
| coupons                       | 5 records    |
| customer_balance_transactions | 82 records   |
| customers                     | 82 records   |
| events                        | 873 records  |
| invoice_items                 | 376 records  |
| invoice_line_items            | 6013 records |
| invoices                      | 5400 records |
| payment_intents               | 1329 records |
| payouts                       | 219 records  |
| plans                         | 14 records   |
| promotion_codes               | 2 records    |
| refunds                       | 4 records    |
| subscription_items            | 1995 records |
| subscriptions                 | 1686 records |
| transfers                     | 1 records    |


Empty streams:
| Streams                        | Records      |
| :------------------------------|:-------------|
| external_account_bank_accounts | 0 records    |
| products                       | 0 records    |

**Comment by roman-yermilov-gl** [comment]
It does not mean we need so many records, but we need at least few of them

**Comment by igrankova** [comment]
Can't fill the streams with data:
bank_accounts
external_account_bank_accounts
checkout_sessions
checkout_sessions_line_items
transfers
Creating a separate task for this: https://github.com/airbytehq/airbyte/issues/22363

**Comment by lazebnyi** [comment]
@roman-yermilov-gl it seems we unblock for now

**Comment by lazebnyi** [comment]
follow up ticket -  #22363
